src/deployment/azuredeploy.bicep

```diff
@@ -164,7 +167,7 @@ module eventGrid 'bicep-templates/event-grid.bicep' = {
 
 // try to make role assignments to deploy as late as possible in order to has principalId ready
 resource roleAssigmentsPy 'Microsoft.Authorization/roleAssignments@2020-10-01-preview' = [for r in roleAssignmentsParams: {
-  name: guid('${resourceGroup().id}${r.suffix}')
+  name: guid('${new_guid}${r.suffix}')
```

Comment:
name: newGuid() 

?

Reply:
newGuid() can't be used outside of a param it seems, so this was a workaround:
`Function "newGuid" is not valid at this location. It can only be used as a parameter default value. bicep(BCP065)`